odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1203,9 +1203,14 @@ def _prepare_qcontext(self):
         """ Returns the qcontext : rendering context with website specific value (required
             to render website layout template)
         """
+        user_id = self.env["res.users"].browse(self.env.user.id)
```

Comment:
This method is used by anyone tryng to render a view. I don't think we wanna push those info at this point.

Rather, look into `webclient_rendering_context` (or the v13.0 equivalent if different) to render the webclient_bootstrap template with the right icon

the website module already has that kind of mechanism
`        <t t-set="x_icon" t-value="website.image_url(website, 'favicon')"/>`